app/assets/stylesheets/govuk-component/_tasklist.scss

```diff
@@ -0,0 +1,358 @@
+.pub-c-tasklist {
+  position: relative;
+  margin-top: $gutter-half;
+  margin-left: $gutter-half;
+  margin-bottom: $gutter;
+}
+
+// scss-lint:disable SelectorFormat
+
+// when js is enabled
```

Comment:
You could do `.js-enabled …` rather than use the comment.

Reply:
I could, yes. I wonder why it wasn't that way in the original accordion? All I'm doing is changing the class name.